frontend/src/views/course/index.vue

```diff
@@ -71,19 +71,18 @@ export default {
     }),
     ratingsExist() {
       return this.courseRatings && this.courseRatings[0].value >= 0
+    },
+    recommend() {
+      return this.courseRatings.find(rating => rating.text === 'Recommended')
     }
   },
   created () {
     this.$store.dispatch('course/getCourse', this.code)
   },
   beforeRouteUpdate ({ params: { code } }, from, next) {
-    // called when the route that renders this component has changed,
-    // but this component is reused in the new route.
-    // For example, for a route with dynamic params `/foo/:code`, when we
-    // navigate between `/foo/1` and `/foo/2`, the same `Foo` component instance
-    // will be reused, and this hook will be called when that happens.
-    // has access to `this` component instance.
-    if (this.code && this.code !== code) {
+    // BUG as far as I'm aware this is only called
```

Comment:
i think this fixed a bug where data was sticking between coursesss... but if that's happening again...

Reply:
Doesn’t seem to be doing anything.

Reply:
The data isn’t sticking anyway; it’s the api that’s not wkrking